cmd/kubeapps-apis/buf.gen.yaml

```diff
@@ -12,3 +12,14 @@ plugins:
     out: gen
     opt:
       - module=github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen
+  # Consider migrating to openapiv3 once ready
+  # https://github.com/grpc-ecosystem/grpc-gateway/pull/2147
+  - name: openapiv2
+    out: docs
+    strategy: all
```

Comment:
It was the key point for `allow_merge` to work...
For the record, the default value is  `directory`, so the merge logic, in each folder, was overwriting the file again and again.
Anyway, it was worthwhile as I have learned a bit more about buf :P

Reply:
Nice - so we can combine/merge the schema docs without needing the (npm-based) swagger-combine? Excellent if so!